src/pages/PolicyPage/PolicyPage.tsx

```diff
@@ -0,0 +1,224 @@
+            </section>
+            <section>
+              <h6>According to CalOPPA, we agree to the following:</h6>
+              <p>Users can visit our site anonymously.</p>
+              <p>
+                Once this privacy policy is created, we will add a link to it on
+                our home page or as a minimum, on the first significant page
+                after entering our website.
+              </p>
+              <p>
+                Our Privacy Policy link includes the word &apos;Privacy&apos;
+                and can easily be found on the page specified above.
+              </p>
+              <ul>
+                You will be notified of any Privacy Policy changes:
+                <li>On our Privacy Policy Page</li>
+              </ul>
+              <ul>
+                Can change your personal information:
+                <li> By emailing us</li>
+              </ul>
+            </section>
+            <section>
+                <h6>COPPA (Children Online Privacy Protection Act)</h6>
+                <p>When it comes to the collection of personal information from children under the age of 13 years old, the Children&apos;s Online Privacy Protection Act (COPPA) puts parents in control. The Federal Trade Commission, United States&apos; consumer protection agency, enforces the COPPA Rule, which spells out what operators of websites and online services must do to protect children&apos;s privacy and safety online.</p>
+            </section>
+            <section>
+                <h6>CAN SPAM Act</h6>
+                <p>The CAN-SPAM Act is a law that sets the rules for commercial email, establishes requirements for commercial messages, gives recipients the right to have emails stopped from being sent to them, and spells out tough penalties for violations.</p>
+                <ul>
+                    <li>Send information, respond to inquiries, and/or other requests or questions</li>
+                    <li>Process orders and to send information and updates pertaining to orders.</li>
+                </ul>
+                <ul>
+                    <li>Not use false or misleading subjects or email addresses.</li>
+                    <li>Identify the message as an advertisement in some reasonable way.</li>
+                    <li>Include the physical address of our business or site headquarters.</li>
+                    <li>Monitor third-party email marketing services for compliance, if one is used.</li>
+                    <li> Honor opt-out/unsubscribe requests quickly.</li>
+                    <li>Allow users to unsubscribe by using the link at the bottom of each email.</li>
+                </ul>
+            </section>
+            <section>
+                <h6>Contacting Us</h6>
+                <p>If there are any questions regarding this privacy policy, you may contact us using the information below.</p>
+                <a href="https://globalmusicinstitute.in/">https://globalmusicinstitute.in/</a>
```

Comment:
change the href to our default "/"

Reply:
@pashedin added the  home page here with Link to